flutter/example/integration_test/integration_test.dart

```diff
@@ -0,0 +1,106 @@
+import 'package:flutter/widgets.dart';
+import 'package:flutter_test/flutter_test.dart';
+import 'package:integration_test/integration_test.dart';
+import 'package:sentry_flutter/sentry_flutter.dart';
+import 'package:sentry_flutter_example/main.dart';
+
+void main() {
```

Comment:
We need a start transaction/finish in order to trigger `fetchNativeAppStart`, `beginNativeFrames`, etc.